VoxelisedSimulation/SubScripts/EstimateScatter.sh

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
best to start with `#!/usr/bin/env bash`, see https://bash.cyberciti.biz/guide/Shebang. As you use `trap ...ERROR` below, you depend on `bash` anyway

Reply:
Not ideal, this is used in other scripts too. However it is such a useful feature. I will raise it as an issue post merge.

Reply:
sure. by the way, I saw you use `sh script` somewhere, maybe the doc? That would fail if it's a `bash` script.